lib/webdrivers/chromedriver.rb

```diff
@@ -14,7 +14,11 @@ def current_version
 
       def latest_version
         raise StandardError, "Can not reach site" unless site_available?
-        Gem::Version.new(get(URI.join(base_url, "LATEST_RELEASE")))
+
+        # Latest webdriver release for installed Chrome version
+        release_file     = "LATEST_RELEASE_#{chrome_major_version}"
```

Comment:
Not sure if it matters, but Chromedriver's suggestion is to use the first three numbers of the version:

> Take the Chrome version number, remove the last part, and append the result to URL "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_". For example, with Chrome version 72.0.3626.81, you'd get a URL "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_72.0.3626".

Just using the major version was only mentioned if using a non-stable/beta Chrome release. Again, I'm not sure how much it matters.

Reply:
Thanks for pointing that out. I have renamed `#chrome_major_version` to `#release_version` - it uses the first three version numbers now.